js/src/SmartConnect/index.d.ts

```diff
@@ -0,0 +1,58 @@
+import WebsocketConnection from '../WebsocketConneciot';
+
+export declare const DEFAULT_SESSION_MANAGER_URL: string;
+
+export interface ISmartConnectInitialValues {
+  config: {
+    // URL to connect to. E.g., "ws://localhost:1234".
+    sessionURL?: string;
+    // Defaults to  SESSION_MANAGER_URL.
```

Comment:
This is the end point of the launcher who's responsible to start the process for you.